src/commands/daily.ts

```
@@ -90,7 +94,19 @@
 						})),
 						totals: createTotalsObject(totals),
 					};
-			log(JSON.stringify(jsonOutput, null, 2));
+
+			// Process with jq if specified
+			if (ctx.values.jq != null) {
```

ESLint

[error] Unsafe member access .values on an `any` value.

(ts/no-unsafe-member-access)